### PR TITLE
Theme changes - all mentions of 2023 replaced with 2024

### PR DESCRIPTION
**Pull request by Diplo2by**
1. added new logo in the header. 
2. Sponsor us anchor tags link to the new sponsorship blogpost
3. Changed calendar element to show the new dates on the reminder

### CODE REVIEW

File: footer.php [hunk 1]
```diff
@@ -10,7 +10,7 @@
           <li><a target="_blank" href="https://discord.gg/MxgfAMXwFS">Discord</a></li>
           <li><a target="_blank" href="https://t.me/BarcampBangalore">Telegram</a></li>
           <li><a rel="me" target="_blank" href="https://hachyderm.io/@barcampbangalore">Mastodon</a></li>
-          <li><a target="_blank" href="https://twitter.com/barcampbng">Twitter</a></li>
```
Comment (the100rabh) [inline footer.php:13]: Make it Twitter/X instead please

File: header.php [hunk 2]
```diff
@@ -34,7 +34,7 @@
         <div class="row">
           <div class="col-12" style="text-align: center;">
           <a class="navbar-brand" href="<?php echo home_url( '/' ); ?>">
-            <img class="logo-on-top" src="<?php bloginfo('template_url'); ?>/images/logo-bcb2023.png" alt="test">
+            <img class="logo-on-top" src="<?php bloginfo('template_url'); ?>/images/logo-bcb-no-year.png" alt="test">
```
Comment (the100rabh) [inline header.php:37]: please add  a year logo here

**Review by the100rabh — CHANGES_REQUESTED** [review]
Please resolve the minor comments

File: footer.php (hunk 1 above)
Comment (Diplo2by) [reply footer.php:13]: Done.

**Pull request merged**